### DISCUSSION TITLE
CLI Output

### DISCUSSION THREAD

**Discussion by andymac4182** [markdown-confluence/markdown-confluence#267] [Ideas]
What do people expect the CLI to output?

**Comment by cwmccann** [comment]
I would expect something like:
```
Updating doc1 at http://xxx.atlassian.net/wiki/some/path/doc1
Updating doc2 at http://xxx.atlassian.net/wiki/some/path/doc2
Warning doc3 at http://xxx.atlassian.net/wiki/some/path/doc3 cannot be found.  Was it deleted in confluence? Use --force to recreate this document.
Creating doc4 at http://xxx.atlassian.net/wiki/some/path/doc4
```

Or something like that.  A list of things that it did and a list of warnings or errors and some hint on how to fix it.

**Comment by andymac4182** [comment]
Here is an example of what I have added 
<img width="1222" alt="image" src="https://github.com/markdown-confluence/markdown-confluence/assets/1297073/209b9def-53cc-4ae0-80fa-64da027d1e20">

**Comment by cwmccann** [comment]
Thanks! I tried the latest client and got something similar.

I also got the puppeteer chrome instance to work.  It wasn't straightforward and the installation failed, but by deleting the cached chrome instance it ended up working.

**Comment by andymac4182** [comment]
Glad that helps :)

**Comment by andymac4182** [comment]
Closing off for now. If people have more thoughts please open a new discussion.
